planner_cspace: test_navigate/NavigateWithLocalMap sometimes failing

https://travis-ci.org/github/at-wat/neonavigation/jobs/683120045
```
build_isolated/planner_cspace/test_results/planner_cspace/rosunit-test_navigate.xml: 6 tests, 0 errors, 2 failures, 0 skipped
Summary: 76 tests, 0 errors, 2 failures, 0 skipped
```
```
Built target _run_tests_planner_cspace_gtest_test_motion_cache
Scanning dependencies of target _run_tests_planner_cspace_roslint
Built target _run_tests_planner_cspace_roslint
Scanning dependencies of target _run_tests_planner_cspace_gtest
Built target _run_tests_planner_cspace_gtest
Navagation success.
[33m[ WARN] [1588626926.604572000]: Patrol finished. Waiting next path.[0m
Costmap received.
Map received.
Local map received.
Map applied.
[33m[ WARN] [1588626927.531767000]: Forgetting remembered costmap.[0m
Local map applied.
[31m[ERROR] [1588626927.574897000]: Position jumped (2.370/1.000, 1.556/1.500); clearing history[0m
Costmap received.
Map received.
Local map received.
Costmap received.
[31m[ERROR] [1588626948.483676000]: Position jumped (1.455/1.000, -1.570/1.500); clearing history[0m
Map applied.
[33m[ WARN] [1588626948.545048000]: Forgetting remembered costmap.[0m
Costmap received.
[31m[ERROR] [1588626948.559206000]: Given start or goal is in Rock. (start: 0, end: 100)[0m
... logging to /home/builder/.ros/log/rostest-5f6fb53df17e-21039.log
[ROSUNIT] Outputting test results to /aports/noetic/planner_cspace/abuild/build_isolated/planner_cspace/test_results/planner_cspace/rostest-test_test_navigation_rostest.xml
[Testcase: testtest_navigate] ... ok

[ROSTEST]-----------------------------------------------------------------------

[planner_cspace.rosunit-test_navigate/Navigate][passed]
[planner_cspace.rosunit-test_navigate/NavigateWithLocalMap][FAILURE]------------
/aports/noetic/planner_cspace/abuild/src/planner_cspace/test/src/test_navigate.cpp:272
Expected: (map_local_->data[addr]) != (100), actual: 'd' (100, 0x64) vs 100
--------------------------------------------------------------------------------

[planner_cspace.rosunit-test_navigate/GlobalPlan][passed]

SUMMARY
[1;31m * RESULT: FAIL[0m
 * TESTS: 3
 * ERRORS: 0
[1;31m * FAILURES: 1[0m
```

--- a/planner_cspace/test/src/test_navigate.cpp
+++ b/planner_cspace/test/src/test_navigate.cpp
@@ -104,6 +104,8 @@ protected:
     std_srvs::EmptyRequest req;
     std_srvs::EmptyResponse res;
     srv_forget_.call(req, res);
+
+    ros::Duration(3.0).sleep();
   }
   void cbCostmap(const costmap_cspace_msgs::CSpace3D::ConstPtr& msg)
   {

